target_ws_pkgs.repos

```diff
@@ -97,12 +99,11 @@ repositories:
   uros/micro_ros_utilities:
     type: git
     url: https://github.com/micro-ROS/micro_ros_utilities.git
-    # three commits past 4.0.1
-    version: 74f7a6b08690079ecbb0262db563aa96f667277d
+    version: 4.0.2
   uros/rmw-microxrcedds:
     type: git
     url: https://github.com/micro-ROS/rmw-microxrcedds.git
-    version: 4.0.1
+    version: 4.0.2
```

Comment:
Also suggest to revert this: `4.0.2` only updates Doxygen config and some minor content changes.